src/config.rs

```diff
@@ -1044,6 +1045,7 @@ impl Default for TiKvConfig {
             log_level: slog::Level::Info,
             log_file: "".to_owned(),
             log_rotation_timespan: ReadableDuration::hours(24),
+            panic_when_key_exceed_bound: false,
```

Comment:
Should the default value be true?

Reply:
No, we will set it true in our internal testings, but in user environment we should alert not panic.